### ISSUE TITLE
no-conditional-in-test warns for condition in test.skip

### ISSUE THREAD

**Issue by cassus** [playwright-community/eslint-plugin-playwright#78]
```ts
test.skip(
      ({ baseURL }) => (baseURL || "").includes("localhost"),
      "dev server displays debug information instead of the normal 500 error page",
    )
```

![errorPages_spec_ts_—_org-mat-viz](https://user-images.githubusercontent.com/316826/184884702-911a78b4-48d4-4d30-bfa1-6b245380b4c9.jpg)

Since this is already a conditional expression having `||` in it should be fine by my logic.

**Comment by mskelton** [comment]
Duplicate of #76
